src/antsibull/data/ansible-readme.rst

```diff
@@ -14,6 +14,19 @@ deployment, cloud provisioning, ad-hoc task execution, network automation, and m
 orchestration. Ansible makes complex changes like zero-downtime rolling updates with load balancers
 easy. More information on the Ansible `website <https://ansible.com/>`_.
 
+This is the ``ansible`` community package.
+``ansible`` contains a set of independent Ansible collections that are currated
+by the community, and it pulls in
```

Comment:
```suggestion
by the community and includes
```

Reply:
It doesn't include ansible-core. ansible-core is a separate package. I'm fine with something other than `pulls in`, but I don't like `include`.